src/main/webapp/timestamps.js

```diff
@@ -42,11 +41,8 @@ function timestampToSeconds(timestampMs) {
 }
 
 /**
- * @return {number} The {@code date} in seconds.
+ * Converts `seconds` into milliseconds.
  */
-// TODO: Remove this method once the pull request updating
-// Date to long is approved.
-function getDateInSeconds(date) {
-  return date.getSeconds() + date.getMinutes() * MINUTES_TO_SECONDS +
-      date.getHours() * HOURS_TO_SECONDS;
+function secondsToMilliseconds(seconds) {
```

Comment:
nit: maybe rename `seconds` we already know seconds is being passes in by the function name

Reply:
I think I'm going to keep it as `seconds` if that's okay because I want the function to be general.